ext/AvizExt/theme.jl

```diff
@@ -82,4 +85,118 @@ function scenario_colors!(obs_color::Observable, color_map::Vector, scen_types::
+end
+function cluster_colors(clusters::BitVector)::Vector{RGBA{Float32}}
+    if unique(clusters) == [0]
+        return fill(parse(Colorant, :red), length(clusters))
+    elseif unique(clusters) == [1]
+        return fill(parse.(Colorant, :blue), length(clusters))
+    end
+    unique_colors::Vector{RGBA{Float32}} = parse.(Colorant, [:red, :blue])
+    return cluster_colors(Int64.(clusters), unique_colors)
+end
+
+"""
+    cluster_alphas(clusters::Vector{Int64})::Vector{Float64}
+    cluster_alphas(clusters::BitVector)::Vector{Float64}
+
+Get color alphas for each cluster weighted by number of scenarios.
+
+# Arguments
+- `clusters` : Vector with scenario cluster numbers
+
+# Returns
+Vector with one color alpha for each cluster.
+"""
+function cluster_alphas(clusters::Vector{Int64})::Dict{Int64,Float64}
+    alphas::Dict{Int64,Float64} = Dict()
+
+    for (i, cluster) in enumerate((unique(clusters)))
+        n_scens::Int64 = count(clusters .== cluster)
+        base_alpha::Float64 = 1.0 / (n_scens * 0.05)
+        alphas[i] = max(min(base_alpha, 0.6), 0.1)
+    end
+
+    return alphas
+end
+function cluster_alphas(clusters::BitVector)::Dict{Int64,RGBA{Float32}}
+    return cluster_alphas(Int64.(clusters))
+end
+
+"""
+    cluster_labels(clusters::Vector{Int64})::Vector{String}
+    cluster_labels(clusters::BitVector)::Vector{String}
+    cluster_labels(clusters::Vector{Int64}, data::AbstractVector{<:Real})::Vector{String}
+    cluster_labels(clusters::BitVector, data::AbstractVector{<:Real})::Vector{String}
+    cluster_labels(cluster_names::Vector{String}, data::AbstractVector{<:Real})::Vector{String}
+
+Get labels for each cluster.
+
+# Arguments
+- `clusters` : Vector with scenario cluster numbers
+
+# Returns
+Vector of labels for each cluster.
+"""
+function cluster_labels(clusters::Vector{Int64})::Vector{String}
+    return "Cluster " .* string.(unique(clusters))
+end
+function cluster_labels(clusters::BitVector)::Vector{String}
+    return [BINARY_LABELS[cluster] for cluster in unique(clusters)]
+end
+function cluster_labels(
+    clusters::Vector{Int64}, data::AbstractVector{<:Real}
+)::Vector{String}
+    cluster_names::Vector{String} = ["Cluster $(cluster)" for cluster in clusters]
+    return cluster_labels(cluster_names, data)
+end
+function cluster_labels(clusters::BitVector, data::AbstractVector{<:Real})::Vector{String}
+    cluster_names::Vector{String} = [BINARY_LABELS[cluster] for cluster in clusters]
+    return cluster_labels(cluster_names, data)
+end
+function cluster_labels(
+    cluster_names::Vector{String}, data::AbstractVector{<:Real}
+)::Vector{String}
+    legend_labels = Vector{String}(undef, length(unique(cluster_names)))
+    for (idx, name) in enumerate(unique(cluster_names))
+        # TODO Extract to external function and accept as argument
+        cluster_metric = mean(data[name .== cluster_names])
+        metric_formatted = @sprintf "%.1e" cluster_metric
```

Comment:
I'm okay with this for now - I think later when we have metrics and arrays with metadata, we can get rid of this and format according to appropriate units (e.g., convert to km^2 or similar)

Reply:
Absolutely!